src/mug/components/page/styles.clj

```diff
@@ -1,14 +1,13 @@
 (ns mug.components.page.styles)
 
 (def defaults
-  [[:.page {:display  "table"
-            :position "static"
-            :width    "100%"
-            :height   "100%"}]
+  [[:.page {:display "table"
+            :width   "100%"
+            :height  "100%"}]
    [:.page-row {:display "table-row"
                 :width   "100%"}]
+   [:.page-row.header {:height "40px"}]
    [:.page-cell {:display "table-cell"
                  :width   "100%"}]
-   [:.page-content {:background-color "red"
-                    :max-height "100%"
-                    :overflow-y "auto"}]])
+   [:.page-content {:height  "100%"
+                    :padding "15px"}]])                     ; TODO: get rid of this padding
```

Comment:
Any reason for more than one space before `;`?

Reply:
Done